x/group/keeper/msg_server.go

```diff
@@ -747,20 +747,26 @@ func (k Keeper) Exec(goCtx context.Context, req *group.MsgExec) (*group.MsgExecR
 	var logs string
 	if proposal.Status == group.PROPOSAL_STATUS_ACCEPTED && proposal.ExecutorResult != group.PROPOSAL_EXECUTOR_RESULT_SUCCESS {
 		// Caching context so that we don't update the store in case of failure.
-		ctx, flush := ctx.CacheContext()
+		cacheCtx, flush := ctx.CacheContext()
 
 		addr, err := sdk.AccAddressFromBech32(policyInfo.Address)
 		if err != nil {
 			return nil, err
 		}
-		_, err = k.doExecuteMsgs(ctx, k.router, proposal, addr)
+
+		results, err := k.doExecuteMsgs(cacheCtx, k.router, proposal, addr)
 		if err != nil {
 			proposal.ExecutorResult = group.PROPOSAL_EXECUTOR_RESULT_FAILURE
 			logs = fmt.Sprintf("proposal execution failed on proposal %d, because of error %s", id, err.Error())
-			k.Logger(ctx).Info("proposal execution failed", "cause", err, "proposalID", id)
+			k.Logger(cacheCtx).Info("proposal execution failed", "cause", err, "proposalID", id)
```

Comment:
Should the logger use the regular context? Are these being logged?

Reply:
Hmm, perhaps? It was previously using the cached context as `ctx` was being reassigned above.
I'm happy to change it to use current context